tests/components/multiselect/multiselect.test.jsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper.jsx';
+import MultiSelect from 'components/multiselect/multiselect.jsx';
+
+describe('components/multiselect/multiselect', () => {
+    const totalCount = 80;
+    const optionsNumber = 80;
+    const users = [];
+    for (var i = 0; i < optionsNumber; i++) {
+        users.push({id: i, value: i});
+    }
+
+    const baseProps = {
+        users,
+        totalCount,
+        values: [{id: 100, value: 100}],
+        saving: false,
+        options: users,
+        perPage: 50,
```

Comment:
Snapshot is too big that it's potentially hard to compare change in the future.  Maybe reduce the size by dividing the counts to 10.  Could be: perPage: 5, totalCount: 8, optionsNumber: 8

Reply:
Ya true. Will change them.